README.md

```diff
@@ -190,33 +190,35 @@ Now we understand the basic concepts, we get into a few more important details.
 
 ## Equity Compensation Details
 
+In this section we'll lay out the differences, benefits, and drawbacks of common types of granted equity. Both **stock options** and **restricted stock units (RSUs)** are common forms of equity compensation, with RSUs more common for larger companies and options more common for startups.
+
 ### Stock options
 
-- [**Stock options**](https://en.wikipedia.org/wiki/Employee_stock_option) (called “employee stock options” when given to employees) are contracts that allow you to buy shares. When you buy shares, it’s called **exercising** the options. Options are not the same as stock; they are only the right to buy stock upon and subject to the conditions specified in the option agreement.
-- Stock options allow you to buy shares at a fixed price per share, the **strike price**. The strike price is generally set lower (often much lower) than what people expect will be the future value of the stock, which means you can make money when you sell the stock.
-- Options expire. You need to know how long the **exercise window** will be open.
-  - Options are only exercisable for a fixed period of time, typically seven to ten years as long as you are working for the company.
-  - ❗ Importantly, options can expire after you quit working for the company. Often the expiration is **90 days** after termination of service, making the options effectively worthless if you cannot exercise before that point.
-  - 🔹 Recently (since around 2015) a few companies are finding ways to keep the exercise window open for years after leaving a company, and promote this as fairer to employees. See [this list](https://github.com/holman/extended-exercise-windows), which includes
+- [**Stock options**](https://en.wikipedia.org/wiki/Employee_stock_option) (“employee stock options,” in this case) are contracts that allow you to buy shares in the company you work for. When you buy shares, you are **exercising** the options. Options are not the same as stock; they are only the right to buy stock upon and subject to the conditions specified in the option agreement.
+- Stock options allow you to buy shares at a *fixed* price per share, the **strike price**. The strike price is generally set lower (often much lower) than what people expect will be the future value of the stock, which means you can make money when you sell the stock. 
+
+Sounds great, right? But wait just a minute. Let's get into some of those conditions and limiations on your option agreement we mentioned above:
```

Comment:
In terms of formatting structure, I'm not sure about a bulleted list covering the basics, a paragraph interlude, and then another bulleted list that elaborates on the interlude. If we add the interlude I'll remove the bullets on the first two points.